UPGRADE.md

```diff
@@ -1,5 +1,34 @@
 # Upgrade
 
+## master
+
+### RouteProvider
+
+The method `Sulu\Bundle\RouteBundle\Routing\RouteProvider::__construct` has removed the second parameter.
```

Comment:
Why add a breaking change when not really needed? I would let the service be injected and remove it on dev-develop / 2.0 and add there the breaking change.